projects/core/src/cart/facade/cart-data.service.ts

```diff
@@ -9,7 +14,24 @@ export class CartDataService {
   private _cart: Cart;
   private _getDetails = false;
 
-  constructor() {}
+  constructor(
+    protected store: Store<StateWithCart>,
+    protected authService: AuthService
+  ) {
```

Comment:
Is there any other way to avoid long-living subscrptions in the constructor?

Reply:
Wanted this service to be read only. Decided to keep logic responsible for setting this values in service.

Additionally removed everything related to `details` as those data should be passed to effect from facade.